src/CC/ShellCheck/Analyze.hs

```diff
@@ -26,7 +24,11 @@ analyze env path = do
     checkSpec x = emptyCheckSpec { csFilename = path, csScript = x }
 
     interface :: SystemInterface IO
-    interface = SystemInterface { siReadFile = defaultInterface }
+    interface = SystemInterface
+        { siReadFile = defaultInterface
+        , siFindSource = error "TODO"
+        , siGetConfig = error "TODO"
```

Comment:
@mcfilib I meant to ask about these. I don't know what these fields are used for, or if these `errors` are safe. Do you know?

Reply:
@pbrisbin in all honesty, i'm not sure because i haven't kept up to date with `ShellCheck` in some time. obviously it'd be good not to have these `errors` here at all. 

i'm at the point where i don't really have time to maintain this project but you're very welcome to investigate or, if you'd be interested, i'd be happy to transfer ownership of the repository over to you.

let me know what you think.